docs/snippets/all/archetypes/image_send_columns.py

```diff
@@ -17,16 +17,16 @@
 
 # Log the ImageFormat and indicator once, as static.
 format_static = rr.components.ImageFormat(width=width, height=height, color_model="RGB", channel_datatype="U8")
-rr.log("images", [format_static, rr.Image.indicator()], static=True)
+rr.send_columns("images", indexes=[], columns=rr.Image.columns(format=format_static))
```

Comment:
Why not this here?

```suggestion
rr.log("images", rr.Image.update_fields(format=format_static), static=True)
```

Reply:
So we can re-enable comparisons of the snippets between the 3 languages (mixing logging and sending has a non-deterministic order).